### ISSUE TITLE
VHDL modules using Verilog libraries - library not found

### ISSUE THREAD

**Issue by rameloni** [TerosTechnology/vscode-terosHDL#687]
Hello, I have a VHDL module that imports and uses a Verilog library. Even though I setup the project properly, Teros is not able to recognize the verilog libraries and modules.

How can I do that?

**Comment by qarlosalberto** [comment]
Sadly it's not possible. TerosHDL recognizes VHDL and Verilog/SV independently.

**Comment by rameloni** [comment]
Alright, thank you! Is there an idea of supporting it in Teros?

**Comment by qarlosalberto** [comment]
Not in the near future, it's s big task.

**Comment by rameloni** [comment]
I see I understand. Apart from that, I'm also facing an issue with vendors IPs. For example, when I try to import a Xilinx library teros reports "not found". 
I'm mainly using teros for its linting and checking features. Maybe, it would be nice to mark something as "blackbox" or simply ask teros to ignore it. What do you think?

**Comment by qarlosalberto** [comment]
Yes, I'm agree. Can you create an issue for that?

**Comment by rameloni** [comment]
Yes https://github.com/TerosTechnology/vscode-terosHDL/issues/688
